modules/storage-api/src/testFixtures/java/org/apache/ignite/internal/storage/BaseMvStoragesTest.java

```diff
@@ -181,7 +158,7 @@ protected static List<IgniteBiTuple<TestKey, TestValue>> drainToList(Cursor<Read
         }
     }
 
-    protected final void assertRowMatches(TableRow rowUnderQuestion, TableRow expectedRow) {
+    protected final void assertRowMatches(@Nullable TableRow rowUnderQuestion, TableRow expectedRow) {
```

Comment:
Null is a valid value, because it can be passed as an argument here. Why do we perform null-check otherwise?

Reply:
At the discretion of Alexander.

Reply:
Null is not a valid value if you throw an exception when this value is passed. `@Nullable` parameter means that `null` can be accepted by a function in a meaningful way.

Reply:
```
By convention, this annotation applied only when the value should always be checked against {@code null} because the developer could do nothing to prevent null from happening.
```
It can be accepted in a meaningful way. It means, that there's an error.

Reply:
Having `null` here is not an error in program (our test), it's an error in the tested program, isn't it?

Reply:
mkay, so this is a part of the test then